app/v2/controllers/AmendForeignPropertyPeriodSummaryController.scala

```diff
@@ -16,18 +16,17 @@
 
 package v2.controllers
 
-import api.controllers.{AuthorisedController, BaseController, EndpointLogContext}
+import api.controllers._
 import api.hateoas.HateoasFactory
 import api.models.audit.{AuditEvent, AuditResponse, GenericAuditDetail}
+import api.models.auth.UserDetails
 import api.models.errors._
 import api.services.{AuditService, EnrolmentsAuthService, MtdIdLookupService}
-import cats.data.EitherT
-import cats.implicits._
-import play.api.libs.json.{Json, JsValue}
+import play.api.libs.json.{JsValue}
 import play.api.mvc.{Action, ControllerComponents}
 import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.play.audit.http.connector.AuditResult
-import utils.{IdGenerator, Logging}
+import utils.{IdGenerator}
```

Comment:
(applies for line 25 as well) we have single imports with unnecessary brackets around them i.e. should be `import utils.IdGenerator`